src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/FixAllContextHelper.cs

```diff
@@ -50,6 +54,49 @@ internal static class FixAllContextHelper
 
                     break;
 
+                case FixAllScope.ContainingMember or FixAllScope.ContainingType:
+                    if (document != null && !await document.IsGeneratedCodeAsync(cancellationToken).ConfigureAwait(false))
```

Comment:
doc why the IsGeneratedCode check.  i'm not getting why that is here.

Reply:
This check already existed for `FixAllScope.Document` above, so I just retained it to match behavior for partial type declarations in generated code: https://github.com/dotnet/roslyn/blob/7bb079e5ceb700d1772b88f7f1b798807fe31805/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/FixAllContextHelper.cs#L44-L45

I believe @sharwell added this check to ensure that FixAll operations don't end up changing generated code.